File: php/examples/get_resource_page.php
<?php
require_once 'base_request.php';

// Example of GET /resources/:id/page/:page

$debug = true;
$resource_id = $_ENV['AGENCY_RESOURCE_ID'];
$page = $_ENV['PAGE_NUMBER'];
$request_data = '';
$request_params = [];
$url_path = '/resources/' . $resource_id . '/page/' . $page;

send_request('GET', $url_path, $request_params, $request_data);

File: csharp/examples/GetResourcePage.cs
using System;
namespace Examples.examples
{
    // Example of GET /resources/:resource_id/page/:page_number

    public class GetResourcePage : BaseRequest
	{
        private string _resourceId;
        private string _pageNumber;

        public GetResourcePage()
		{
            _pageNumber = Environment.GetEnvironmentVariable("PAGE_NUMBER");
            _resourceId = Environment.GetEnvironmentVariable("AGENCY_RESOURCE_ID");
            Console.WriteLine($"GetResource: _resourceId={_resourceId} _pageNumber={_pageNumber}");
        }

        public override string SendRequest()
        {
            return GetRequest($"resources/{_resourceId}/page/{_pageNumber}");
        }
    }
}